wrapper/symbols/crypto_pwhash_str_needs_rehash.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "crypto_pwhash_str_needs_rehash",
+    "type": "function",
+    "inputs": [
+            {
+                    "name": "hashed_password",
+                    "type": "string"
+            },
+            {
+                    "name": "opsLimit",
+                    "type": "uint"
+            },
+            {
+                    "name": "memLimit",
+                    "type": "uint"
+            }
+    ],
+    "target": "var result = libsodium._crypto_pwhash_str_needs_rehash(hashed_password_address, opsLimit, 0, memLimit) | 0",
```

Comment:
Is this right? It is not totally clear to me why some of these wrappers have a `0` in their argument list. It seemed like they follow `unsigned long long`s so I put one here.

Reply:
Yep, looks fine!